docs/sbt.md

```
@@ -0,0 +1,45 @@
+// project/plugins.sbt
+addSbtPlugin("net.exoego.uika" % "sbt-uika" % "VERSION_PLACEHOLDER")
+```
+
+```scala
+// build.sbt — optional: gate only on reachable violations, and suppress known false positives.
+ThisBuild / uikaFailOn := "reachable"
+ThisBuild / uikaExcludeFiles := Seq(baseDirectory.value / "uika-exclude.toml")
+```
+
+```console
+$ sbt uikaDumpClasspath   # writes target/uika/classpath.json (override via the uikaOutput setting)
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)